src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -50,7 +50,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
       <% if (request.getSession().getAttribute("user") != null) { %>
-    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <a href="/allusers">All Users</a>
```

Comment:
I think rather than changing the "hello user" link to all users, it is more user friendly to create a new hyperlink for all users. My thought is to either leaving the "hello user" link as it is and linking it to the user's own bio page, or changing it to "logout user". Either way, at least users know clearly that they are logged in.